src/Propulsion.MessageDb/ReaderCheckpoint.fs

```diff
@@ -0,0 +1,81 @@
+module Propulsion.MessageDb.ReaderCheckpoint
+
+open Npgsql
+open NpgsqlTypes
+open Propulsion.Feed
+open Propulsion.Infrastructure
+open System
+
+let createIfNotExists (conn : NpgsqlConnection, schema: string) =
+    let cmd = conn.CreateCommand()
+    cmd.CommandText <- $"
+      create table if not exists {schema}.propulsion_checkpoints (
+        stream_name text not null,
+        consumer_group text not null,
+        global_position bigint not null,
```

Comment:
arguable whether this should be `position` as this could technically be used to store checkpoints for any other store

Reply:
yes; well figured - position is better